CONTRIBUTING.md

```diff
@@ -11,8 +11,8 @@ Install a few dev dependencies for `make lint`:
 ```bash
 go install github.com/mgechev/revive@v1.1.3
 go install mvdan.cc/gofumpt@v0.3.1
-go install honnef.co/go/tools/cmd/staticcheck@v0.3.0
-go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0
+go install honnef.co/go/tools/cmd/staticcheck@v0.4.0
+go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.0
```

Comment:
Amazing, thanks!

in fact it's 1.51.2: https://github.com/flashbots/mev-boost/blob/develop/.github/workflows/lint.yml

Reply:
Great! just updated the PR using same CI dependencies